_aicspylibczi/Reader.cpp

```diff
@@ -323,13 +323,23 @@ namespace pylibczi {
   void
   Reader::addOrderMapping()
   {
+      bool firstGo = true;
+      bool inconsistent = false;
       // create a reference for finding one or more subblock indices from a CDimCoordinate
       m_czireader->EnumerateSubBlocks([&](int index_, const libCZI::SubBlockInfo& info_) -> bool {
           if (isPyramid0(info_)) {
               m_orderMapping.emplace_back(std::piecewise_construct,
                   std::make_tuple(&(info_.coordinate), info_.mIndex, isMosaic()),
                   std::make_tuple(index_)
               );
+              if( firstGo ) m_pixelType = info_.pixelType;
```

Comment:
Adding a linter would catch the `if(` I think a space is supposed to be between the if and paren.

Reply:
I also prefer bracketing all `if` predicates but that's a style thing.

Reply:
Do you mean wrapping the entire if block in {}?
```
{
    if( firstGo ) m_pixelType = info_.pixelType; 
}
```

Reply:
I think I misused the word predicate.  I looked this up and the term for the "then" part of an if statement is called the "consequent" or apodosis!   Anyway, big words but I just meant that I like to always scope the "then" parts, after the condition, with brackets just to be on the safe side.

Reply:
`if (firstGo) { m_pixelType = info_.pixelType; }`